index.rst

```diff
@@ -1,61 +1,114 @@
-..
-  Technote content.
+:tocdepth: 1
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+.. sectnum::
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Abstract
+========
 
-  Use the following syntax for sections:
+Butler manages read and write access to Rubin Observatory data.
+Different types of data require different access control rules (Data Release versus User Generated data products, for example).
+Butler itself also needs a backing SQL database to store its information, which should be protected from unauthorized modification.
+This tech note proposes a design for access control for both Butler queries and for data upload and download mediated by Butler.
 
-  Sections
-  ========
+Current state
+=============
 
-  and
+Butler is a Python library that mediates access to Rubin Observatory data.
+It uses a SQL database to store metadata about files.
+That database can either be a local SQLite database or a remote SQL database.
 
-  Subsections
-  -----------
+When a Rubin Science Platform user wants to process some data, they would call Butler to download the data, perform whatever processing they wish, and then call Butler again to store the results somewhere for future use.
```

Comment:
I wouldn't say "download".  They are just trying to "access" the data in a Python environment.  If the files are in a Posix data store there really is no "downloading" involved; if they are on cloud storage in the IDF, even then I think "download" is suggesting something remote, and that's not currently a requirement.  (It could be a future one, but that's not the present situation.)